templates/registration/login.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <h1>Login page</h1>
+  {% if form.error %}

```

Comment:
The condition `{% if form.error %}` is incorrect. It should be `{% if form.errors %}` to properly check for any form validation errors. The `errors` attribute is a dictionary containing all the errors for the form fields.